src/api/app/controllers/webui/projects/project_configuration_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Webui
+  module Projects
+    class ProjectConfigurationController < WebuiController
+      before_action :set_project
```

Comment:
What about ``verify_authorized``?
https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used